static/app/views/eventsV2/table/queryField.tsx

```diff
@@ -373,6 +409,8 @@ class QueryField extends React.Component<Props> {
             onChange={this.handleFieldParameterChange}
             inFieldLabel={inFieldLabels ? t('Parameter: ') : undefined}
             disabled={disabled}
+            styles={this.FieldSelectStyles}
```

Comment:
Is the inFieldLabels check needed here? `{!inFieldLabels ? this.FieldSelectStyles : undefined}`

Reply:
I don't actually see this in use, but going to add it just to be consistent 👍